.hlint.yaml

```diff
@@ -38,6 +38,8 @@
 - ignore: {name: Redundant if, within: [Language.Plutus.Coordination.Contracts.CrowdFunding, Language.Plutus.Coordination.Contracts.Vesting, Language.Plutus.Runtime.TH]}
 # This is rarely better, and often more confusing
 - ignore: {name: Use asks}
+# It seems clearer to use case than fromMaybe
+- ignore: {name: fromMaybe}
```

Comment:
I disagree with this.

Reply:
I'm not sure what's going on here.  The CI checks were failing with hlint saying things like this:
```
Found:
  case IntMap.lookup (unUnique $ nameUnique name) env of
      Nothing -> throw $
                   LMachineStringException
                     ("Name " ++ show (nameString name) ++ " missing from environment")
      Just loc -> loc
Perhaps:
  fromMaybe
    (throw $
       LMachineStringException
         ("Name " ++ show (nameString name) ++ " missing from environment"))
    (IntMap.lookup (unUnique $ nameUnique name) env)
```

But there's almost identical code in the CEK machine, for example, and presumably it wasn't rejecting that.

Reply:
Ah, sorry, I agree that in this case `case` is much clearer. I thought that adding this rule "disables" `fromMaybe` globally, but it seems it's already "disabled". `hlint` is just being silly, that's how it normally works.

Reply:
Please do feel free to keep the `- ignore: {name: fromMaybe}` line.